hedera-node/hedera-schedule-service/src/main/java/com/hedera/node/app/service/schedule/ReadableScheduleStore.java

```diff
@@ -56,11 +56,11 @@ public interface ReadableScheduleStore {
      * @return a {@code List<Schedule>} of entries that have the same hash as the provided schedule
      */
     @Nullable
-    List<Schedule> getByEquality(@NonNull Schedule scheduleToMatch);
+    Schedule getByEquality(@NonNull Schedule scheduleToMatch);
```

Comment:
This should still be a list (even if it's changed to just be ID values).
The "duplication" hash of the schedule is not enough to know if it's a duplicate, and multiple "candidate" duplicate schedules may have the same "duplication" hash value.

Reply:
The reason is explained below.